cms/static/sass/elements/_modal-window.scss

```diff
@@ -757,6 +758,7 @@
     }
 
     // UI: staff lock section
```

Comment:
Is this comment no longer necessary/applicable?

Reply:
I updated it. Should I remove it completely?